phpBB/phpbb/install/module/install_finish/task/populate_migrations.php

```diff
@@ -46,6 +57,15 @@ public function __construct(\phpbb\install\helper\container_factory $container)
 	 */
 	public function run()
 	{
+		if (!$this->config->get('populate_migration_refresh_before', false))
+		{
+			if ($this->config->get_time_remaining() < 1)
```

Comment:
because the installer does not have finer controll over that, so basically it just forces a refresh if the remaining time is smaller than 1.